### PR TITLE
Implementing an Image Classification App in One Minute for TinyMS 0.3.1

### PR DESCRIPTION
**Pull request by xinwenh**
**What type of PR is this?**

> `/kind enhancement`</br>

**What does this PR do / why do we need it**:

This PR updates tinyms0 3.1 realize the document of "realizing one graphic classification application in one minute" and publish it (Chinese version and English version) on the official document website.

**Which issue(s) this PR fixes**:

**Special notes for your reviewers**:

### CODE REVIEW

**Comment by hellowaywewe** [comment]
@xinwenh please update your branch to ensure that the latest content of your branch is consistent with the main branch.

File: docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb [hunk 1]
```diff
@@ -287,26 +320,26 @@
...
   {
    "cell_type": "markdown",
-   "id": "convertible-timer",
+   "id": "40b5e41a",
    "metadata": {},
    "source": [
     "#### 6.2 List servables\n",
     "\n",
-    "使用`list_servables`函数检查当前后端的serving模型"
+    "使用`list_servables`函数检查当前后端的serving模型,如果这边报错，在root下download_dataset('mnist', '/root'),然后从头运行一遍。"
```
Comment (lwdragon) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:336]: Hi,@xinwenh .This path `/root`  is inconsistent with the previous one.

Comment (hellowaywewe) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:336]: Please modify punctuation formatting problems

Comment (xinwenh) [reply docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:336]: ok i will perfect it

Comment (xinwenh) [reply docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:336]: ok i will perfect it

File: docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb [hunk 2]
```diff
@@ -232,40 +226,79 @@
...
+    "出现如上提示说明服务器已启动且正在运行。"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "european-wallpaper",
+   "id": "7fa5fc91",
+   "metadata": {},
+   "source": [
+    "接下来我们需要到菜单栏点击`File`=>`new Notebook`创建一个新的Jupyter的文件，然后继续完成接下来的操作。"
```
Comment (hellowaywewe) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:268]: This is not the latest thing I asked you to fix yesterday

File: docs/en/source/quickstart/quickstart_in_one_minute.ipynb [hunk 3]
```diff
@@ -227,30 +221,69 @@
...
+    "The above prompt indicates that the server is started and running."
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "greenhouse-dutch",
+   "id": "6d74fafa",
+   "metadata": {},
+   "source": [
+    "Next, we need to go to the menu bar and click `File`=>`new Notebook` to create a new Jupyter file, and then continue to complete the next operation"
```
Comment (hellowaywewe) [inline docs/en/source/quickstart/quickstart_in_one_minute.ipynb:268]: This is not the latest thing I asked you to fix yesterday.

**Comment by lwdragon** [comment]
Hi,@xinwenh.  I found that the picture path of your output in Module 6 inference is inconsistent with that of your code. You'd better update the output.

File: docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb [hunk 4]
```diff
@@ -88,18 +82,18 @@
...
   {
    "cell_type": "markdown",
-   "id": "guided-activation",
+   "id": "bdcc1fcd",
    "metadata": {},
    "source": [
     "### 2. 下载数据集\n",
     "\n",
-    "如果根目录下没有创建`mnist`文件夹则MNIST数据集会被自动下载并存放到根目录，如果`mnist`文件夹已经存在于根目录 ，则此步操作会被跳过"
+    "如果根目录下没有创建`mnist`文件夹则MNIST数据集会被自动下载并存放到根目录，如果`mnist`文件夹已经存在于根目录 ，则此步操作会被跳过。"
```
Comment (hellowaywewe) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:90]: `/home/jovyan` is the user home directory not the root directory.

File: docs/en/source/quickstart/quickstart_in_one_minute.ipynb [hunk 5]
```diff
@@ -227,30 +221,70 @@
...
+   "source": [
+    "The above prompt indicates that the server is started and running."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d74fafa",
+   "metadata": {},
+   "source": [
+    "After the server is started. we need to go to the menu bar and click `File`=>`new Notebook` to create a new jupyter file, and then continue to complete the operation of the client."
```
Comment (hellowaywewe) [inline docs/en/source/quickstart/quickstart_in_one_minute.ipynb:268]: The `started` word should be followed by a comma.

File: docs/en/source/quickstart/quickstart_in_one_minute.ipynb [hunk 6]
```diff
@@ -88,7 +82,7 @@
   },
   {
    "cell_type": "markdown",
-   "id": "guided-activation",
+   "id": "11fd53a1",
    "metadata": {},
    "source": [
     "### 2. Download dataset\n",
```
Comment (hellowaywewe) [inline docs/en/source/quickstart/quickstart_in_one_minute.ipynb:88]: /home/jovyan is the user home directory not the root directory.

File: docs/en/source/quickstart/quickstart_in_one_minute.ipynb [hunk 7]
```diff
@@ -227,30 +221,70 @@
...
+    "from tinyms.serving import Server, Client"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "greenhouse-dutch",
+   "id": "a0ecea1c",
    "metadata": {},
    "source": [
     "### 6. Make predictions\n",
```
Comment (hellowaywewe) [inline docs/en/source/quickstart/quickstart_in_one_minute.ipynb:290]: `/home/jovyan` is the user home directory not the root directory.

File: docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb [hunk 8]
```diff
@@ -30,7 +24,7 @@
   {
```
Comment (lwdragon) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:24]: Hi,@xinwenh , please pay attention to the specification of punctuation, sometimes it is `、 `, sometimes `, `, it seems particularly unprofessional

File: docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb [hunk 9]
```diff
@@ -232,81 +226,123 @@
...
-   "id": "european-wallpaper",
+   "id": "7bded060",
    "metadata": {},
    "source": [
     "### 6. 推理\n",
     "\n",
     "#### 6.1 上传图片\n",
     "\n",
-    "用户需要上传一张0~9之间的数字图片作为输入。如果使用命令行终端，可以使用'scp'或者'wget'获取图片，如果使用Jupyter，点击菜单右上方的'Upload'按钮并且选择上传的图片。本教程中使用的图片可以点击[这里](https://ascend-tutorials.obs.cn-north-4.myhuaweicloud.com/tinyms-test-pics/numbers/7.png)进行下载，将图片保存在根目录下，重命名为'7.png'（或者任何用户喜欢的名字）\n",
+    "用户需要上传一张0~9之间的数字图片作为输入。如果使用命令行终端，可以使用'scp'或者'wget'获取图片，如果使用Jupyter，点击菜单右上方的'Upload'按钮并且选择上传的图片。本教程中使用的图片可以点击[这里](https://ascend-tutorials.obs.cn-north-4.myhuaweicloud.com/tinyms-test-pics/numbers/7.png)进行下载，将图片保存在根目录下，重命名为'7.png'（或者任何用户喜欢的名字）。\n",
```
Comment (lwdragon) [inline docs/zh_CN/source/quickstart/quickstart_in_one_minute.ipynb:294]: Hi,@xinwenh.In order to unify the specification, it is recommended to change 'scp' and 'wget' to `scp` and `wget`

**Review by hellowaywewe — APPROVED** [review]
@xinwenh LGTM. Thanks for your contribution.

**Comment by lwdragon** [comment]
Hi,@xinwenh.The best path is to use ``, for example, change /home/jovyan to `/home/jovyan`.

**Review by hellowaywewe — APPROVED** [review]
LGTM

**Review by lwdragon — APPROVED** [review]
LGTM

**Comment by lwdragon** [comment]
@xinwenh LGTM. Thanks for your contribution.

**Pull request merged**